### PR TITLE
add parser code

### PR DESCRIPTION
**Pull request by ardaakcabuyuk**
WHAT I DID
------------
Added JSON parser to parse the data from Perceval.

HOW TO TEST
------------
`python parser.py`

### CODE REVIEW

File: parser.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+commits = {}
+parents = {}
+
+with open('storybookresult.json') as f:
+    for (i, jsonObj) in enumerate(f):
+        info = json.loads(jsonObj)
+        commits[info["data"]["commit"]] = info
+        parents[info["data"]["commit"]] = [commits[parent] for parent in info["data"]["parents"]]
+
+a = parents["d9f1e643a6d867c8b0350b88e44de0dbb078f445"]
```
Comment (elifozerr) [inline parser.py]: Why is the naming like that? It can have a more sensible name.

File: parser.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+i = 0
+while len(a[0]["data"]["parents"]) > 0:
+    id = a[0]["data"]["parents"][0]
+    a = parents[id]
+    print("iteration", i, id)
+    i += 1
+
+print(a)
```
Comment (elifozerr) [inline parser.py]: Can you remove this before merging?

**Review by elifozerr — CHANGES_REQUESTED** [review]

**Comment by elifozerr** [comment]
I suggest you to update the dependencies accordingly to reduce the unnecessary memory overhead.

File: parser.py (hunk 1 above)
Comment (ardaakcabuyuk) [reply parser.py]: I'll replace this with `root = parents["d9f1e643a6d867c8b0350b88e44de0dbb078f445"]`

**Review by elifozerr — APPROVED** [review]

**Pull request merged**